Running tests with `DPNP_TEST_ALL_INT_TYPES=` raised an exception

The bellow command to run dpnp tests causes the issue:
```bash
$ DPNP_TEST_ALL_INT_TYPES= python -m pytest -ra -x --pyargs dpnp.tests
...
/localdisk/work/antonvol/soft/miniforge3/envs/test_issue/lib/python3.12/site-packages/dpnp/tests/test_absolute.py:7: in <module>
    from .helper import (
/localdisk/work/antonvol/soft/miniforge3/envs/test_issue/lib/python3.12/site-packages/dpnp/tests/helper.py:10: in <module>
    from . import config
/localdisk/work/antonvol/soft/miniforge3/envs/test_issue/lib/python3.12/site-packages/dpnp/tests/config.py:3: in <module>
    all_int_types = int(os.getenv("DPNP_TEST_ALL_INT_TYPES", 0))
E   ValueError: invalid literal for int() with base 10: ''
...
```
It seems the way how env variable `DPNP_TEST_ALL_INT_TYPES` is handled in tests needs to be corrected.

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -28,6 +28,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 * Resolved an issue with an incorrect result returned due to missing dependency from the strided kernel on a copy event in `dpnp.erf` [#2378](https://github.com/IntelPython/dpnp/pull/2378)
 * Updated `conda create` commands build and install instructions of `Quick start guide` to avoid a compilation error [#2395](https://github.com/IntelPython/dpnp/pull/2395)
+* Added handling of empty string passed to a test env variable defining data type scope as a `False` value [#2415](https://github.com/IntelPython/dpnp/pull/2415)
 
 
 ## [0.17.0] - 02/26/2025

--- a/dpnp/tests/config.py
+++ b/dpnp/tests/config.py
@@ -1,6 +1,6 @@
 import os
 
-all_int_types = int(os.getenv("DPNP_TEST_ALL_INT_TYPES", 0))
-float16_types = int(os.getenv("DPNP_TEST_FLOAT_16", 0))
-complex_types = int(os.getenv("DPNP_TEST_COMPLEX_TYPES", 0))
-bool_types = int(os.getenv("DPNP_TEST_BOOL_TYPES", 0))
+all_int_types = bool(os.getenv("DPNP_TEST_ALL_INT_TYPES", 0))
+float16_types = bool(os.getenv("DPNP_TEST_FLOAT_16", 0))
+complex_types = bool(os.getenv("DPNP_TEST_COMPLEX_TYPES", 0))
+bool_types = bool(os.getenv("DPNP_TEST_BOOL_TYPES", 0))

